connexion/apps/asynchronous.py

```diff
@@ -63,7 +63,8 @@ def fn(self) -> t.Callable:
     async def __call__(
         self, scope: Scope, receive: Receive, send: Send
     ) -> StarletteResponse:
-        return await self.fn(scope=scope, receive=receive, send=send)
```

Comment:
Just a fix to get the `AsyncApp` working correctly. Not related to enforcing defaults, but necessary to make the example work.